source/_integrations/alarm_control_panel.template.markdown

```diff
@@ -74,7 +74,7 @@ panels:
           description: Name to use in the frontend.
           required: false
           type: string
-          default: Template Alarm Control Panel
+          default: Template Alarm control panel
```

Comment:
```suggestion
          default: Template Alarm Control Panel
```
Default is defined in the core code, so unless its change there it should not change here.